src/main/java/com/_up/megastore/services/mappers/UserMapper.java

```diff
@@ -0,0 +1,32 @@
+package com._up.megastore.services.mappers;
+
+import com._up.megastore.controllers.requests.CreateUserRequest;
+import com._up.megastore.controllers.responses.UserResponse;
+import com._up.megastore.data.model.User;
+import com._up.megastore.data.utilities.PhoneNumberConversion;
+
+public class UserMapper {
+
+  public static UserResponse toUserResponse(User user) {
+    return new UserResponse(
+        user.getUserId(),
+        user.getUsername(),
+        user.getFullName(),
+        user.getEmail(),
+        user.getPhoneNumber(),
+        user.getRole()
+    );
+  }
+
+  public static User toUser(CreateUserRequest createUserRequest) {
+    return User.builder()
+        .email(createUserRequest.email())
+        .fullName(createUserRequest.fullName())
+        .password(createUserRequest.password())
+        .phoneNumber(
+            PhoneNumberConversion.convertPhoneNumber(createUserRequest.phoneNumber())
+        )
+        .username(createUserRequest.username())
+        .build();
+  }
```

Comment:
Teniendo en cuenta que después tenemos que hacer tests, creo que no estaría mal separar la conversión del número de teléfono del mapper y dejarla en un método aparte privado dentro del servicio

```suggestion
  public static User toUser(CreateUserRequest createUserRequest, String convertedPhoneNumber) {
    return User.builder()
        .email(createUserRequest.email())
        .fullName(createUserRequest.fullName())
        .password(createUserRequest.password())
        .phoneNumber(convertedPhoneNumber)
        .username(createUserRequest.username())
        .build();
  }
```
De esta forma podemos testear la función del mapper y la conversión del número de teléfono de forma individual y separada